setup/security/authentication/x509/index.md

```diff
@@ -103,7 +102,7 @@ Encoding with any other OID can be done by editing the `openssl.conf`.
     keyUsage = nonRepudiation, digitalSignature, keyEncipherment
     1.2.840.10070.8.1 = ASN1:UTF8String:spinnaker-example0\nspinnaker-example1
     ```
-    The same rule as in step above is applied for role defention in this [step](#roles-defenition)
+    The same rule as in step above is applied for role defention in the step above
```

Comment:
s/defention/definition/

Also, which steps are you referring to? The way this reads now, the step above has the same rule as the step above.

Reply:
I was trying to add anchor for explanation about `The final line in this file `1.2.840.10070.8.1= ASN1:UTF8String:spinnaker-example0\nspinnaker-example1` is what matters for creating a client certificate with user role information, as anything after `UTF8String:` is encoded inside of the x509 certificate under the given OID.`

Reply:
@dorbin rephrased the explanation, I hope should be more clear